examples/cpp/sslputget/main.cpp

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifdef _MSC_VER
+#include <direct.h>
+#include <windows.h>
+#ifndef PATH_MAX
+#define PATH_MAX MAX_PATH
+#undef max
+#endif
+#else
+#include <unistd.h>
+#endif
+#include <iostream>
+#include <memory>
+#include <string>
+
+#include <geode/Cache.hpp>
+#include <geode/CacheFactory.hpp>
+#include <geode/PoolManager.hpp>
+#include <geode/RegionFactory.hpp>
+#include <geode/RegionShortcut.hpp>
+
+using apache::geode::client::CacheableString;
+using apache::geode::client::CacheFactory;
+using apache::geode::client::RegionShortcut;
+
+std::string myGetcwd() {
+  char buf[PATH_MAX];
+  std::string cwd;
+
+#ifdef _MSC_VER
```

Comment:
If we changed this to something like:
```
#ifdef _MSC_VER
  char buf[MAX_PATH];
  if (_getcwd(buf, MAX_PATH)) {
    cwd = buf;
  }
#else
  char buf[PATH_MAX];
  if (getcwd(buf, PATH_MAX)) {
    cwd = buf;
  }
#endif
```
could we get rid of all the awful _MSC_VER stuff above?

Reply:
Agreed we can make this a little cleaner. (Ultimately we'll go to std::filesystem)